daemon/info.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-func (daemon *Daemon) SystemInfo() (*types.Info, error) {
+func (daemon *Daemon) SystemInfo(includeOperatingSystem bool) (*types.Info, error) {
```

Comment:
That would be too much @thaJeztah here for just a field I think
I agree with @tiborvass `useLegacyOperatingSystemField` sounds good